app/src/main/java/de/westnordost/streetcomplete/quests/shop_type/SpecifyShopType.kt

```diff
@@ -0,0 +1,64 @@
+package de.westnordost.streetcomplete.quests.shop_type
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.meta.LAST_CHECK_DATE_KEYS
+import de.westnordost.streetcomplete.data.meta.SURVEY_MARK_KEY
+import de.westnordost.streetcomplete.data.meta.toCheckDateString
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder
+import de.westnordost.streetcomplete.data.osm.osmquest.OsmFilterQuestType
+import de.westnordost.streetcomplete.ktx.containsAny
+import java.util.*
+
+class SpecifyShopType : OsmFilterQuestType<ShopTypeAnswer>() {
+
+    override val elementFilter = """
+        nodes, ways, relations with (
+             shop = yes
+             and !man_made
+             and !historic
+             and !military
+             and !power
+             and !tourism
+             and !attraction
+             and !amenity
+             and !leisure
+          )
+    """
+    override val commitMessage = "Specify shop type"
+    override val wikiLink = "Key:shop"
+    override val icon = R.drawable.ic_quest_check_shop // TODO nice icon would be nice (reuse graphic from building quest?)
```

Comment:
I tried to make an icon with that, but I wasn't happy with it. I think this icon is fine. The interface is also identical to that other quest.